packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -2956,6 +2973,26 @@ class InputDecoration {
   /// Only one of [error] and [errorText] can be specified.
   final String? errorText;
 
+
+  /// Builds [Widget] that appears below the [InputDecorator.child] and the border.
+  ///
+  /// If non-null, [errorText] will be passed to this builder, and the returned
+  /// widget will be displayed below the [InputDecorator.child] and the border.
+  ///
+  /// When used with [TextFormField] the text will come from [TextFormField.validator],
+  /// otherwise it will come from [InputDecoration.errorText]
```

Comment:
I think this is redundant with the "see also" bullet point below, so I would just remove this.